### PR TITLE
README update: [migration file] documentation fix

### PR DESCRIPTION
**Pull request by tkivisik**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -134,12 +134,12 @@ List all migrations with an indicator (applied,pending) whether is has been appl
 
 Run this command in the directory where all migrations are stored. Use `--migrations` for a different location.
 
-### plan  [path] [|migration file] [--migrations folder]
```
Comment (emicklei) [inline README.md]: the pipe was included to hint that the migration file argument is not needed

Comment (tkivisik) [reply README.md]: Then sorry, my bad :)

Comment (tkivisik) [reply README.md]: I propose to change the documentation and code to match the following convention ( https://developers.google.com/style/code-syntax )

- mandatory argument just as it is
- voluntary argument in square brackets ==> [voluntary]
- mandatory argument with mutually exclusive options in curly brackets separated by a pipe ==> {one|two|three}
- voluntary argument with mutually exclusive options in square brackets separated by a pipe ==> [one|two|three]

Comment (emicklei) [reply README.md]: yes, let's approve it like this

**Pull request closed without merging**